index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"
+      xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html">
+<head>
+    <title>Rolling Stone Website</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Merriweather:wght@300;400&family=Roboto:ital,wght@0,300;0,400;1,300;1,400&display=swap"
+          rel="stylesheet">
+    <style>
+        body {
+            font-family: 'Merriweather Sans', 'Roboto', sans-serif;
+        }
+    </style>
+</head>
+<body>
+<header class="header">
+    <nav id="nav-bar">
+        <ul>
+            <li><a href="#case_studies_project_section"> Case Studies</a></li>
+            <li><a href="#services_section"> Services</a></li> <!--don't forget the jump links to all list items-->
+            <li><a href="#contactform">Contact Us</a></li> <!--don't forget the jump links to all list items-->
+        </ul>
+    </nav>
+    <div class="header-text">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <p>We are here to help you succeed.</p>
+        <button type="button">Let's work together</button>
+    </div>
+    <div class="squiggly-line-image">
+        <img src="/assets/squiggle_line_stripe.svg" alt="white wavy designed lined">
+    </div>
+</header>
+
+<main>
+    <section class="section-introduction">
+
+        <h2>Rekupe is a digital agency in Los Angeles.</h2>
+        <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h2>
+        <h2>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h2>
+        <div class="introduction-link">
+            <p>Have a question?&nbsp</p> <a href="#contactform">Click here&nbsp</a>
+            <p> to contact us. </p>
+        </div>
+    </section>
+    <a id="case_studies_project_section">
+        <section class="projects-container">
+            <div class="inner-container">
+
+                <article class="project-one">
+                    <div class="first-article">
+                        <h3>Rolling Stone</h3>
+                        <img src="/assets/squiggle_line.svg" class="squiggle-line-purple"
+                             alt="small purple squiggle line underlining h2">
+                        <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a
+                            global
+                            icon.</p>
+                        <h5>View Project</h5>
+                    </div>
+                    <span class="image-wrapper">
+        <img class="image-one" src="/assets/work-image-01.jpeg"
```

Comment:
Voor alle afbeeldingen geldt dat ik ze niet kan zien. Als ik het aanpas naar het volgende, dan wel:
```suggestion
        <img class="image-one" src="assets/work-image-01.jpeg"
```